<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QWidget" native="1" name="examiner" />
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Choose axis</string>
        </property>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QRadioButton" name="button_x" >
           <property name="text" >
            <string>&amp;X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="button_y" >
           <property name="text" >
            <string>&amp;Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="button_z" >
           <property name="text" >
            <string>&amp;Z</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="checkbox" >
          <property name="text" >
           <string>Enable &amp;rotation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button" >
          <property name="text" >
           <string>&amp;Change cone color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
